src/pages/commits/ActiveCommits/ChartToggle.tsx

```diff
@@ -44,21 +44,24 @@ export const ChartToggle: React.VFC<{
                 onChange={onChange}
                 value={currentChartType}
                 name="chart-select"
-                className={css`
-                  font-weight: bold;
-                `}
               >
                 <Radio
                   data-cy="cy-chart-absolute-radio"
                   id="chart-radio-absolute"
                   value={ChartTypes.Absolute}
+                  className={css`
+                    margin: ${size.xs} 0;
+                  `}
                 >
                   Absolute Number
                 </Radio>
                 <Radio
                   data-cy="cy-chart-percent-radio"
                   id="chart-radio-percent"
                   value={ChartTypes.Percentage}
+                  className={css`
+                    margin-bottom: ${size.xs};
+                  `}
```

Comment:
For some reason you can't use `styled` on these, they just stop being interact-able if you do... I reproduced this in a code sandbox: https://g5z3l0.csb.app/